packages/driver-adapter-utils/src/underlying-drivers.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * This module provides a map from the official Prisma Driver Adapter names to their underlying driver names.
+ * E.g., `@prisma/adapter-planetscale` -> `@planetscale/database`
+ */
+
+export const underlyingDriverAdaptersMap = {
```

Comment:
I think this module would make more sense as a part of `cli`, not `driver-adapter-utils`:
- `cli` is the only one using it and I don't see that changing anytime soon
- it's weird that generic library with building blocks and abastractions for specific drivers now has a list of all specific adapters hardcoded.